Resources/public/css/views/dashboard/dashboard.css

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-dashboardblockbasedview,
+.ez-view-dashboardblockbasedview * {
```

Comment:
Yes, it is. I want to have all elements inside the dashboard to behave the same way.